pkgs/development/libraries/kde-frameworks/kholidays.nix

```diff
@@ -8,7 +8,7 @@ mkDerivation {
   name = "kholidays";
   meta = {
     license = with lib.licenses; [ gpl2 lgpl21 fdl12 ];
-    maintainers = kdepimTeam;
+    maintainers = lib.maintainers.bkchr;
```

Comment:
The failing meta check is because maintainers is not a list.
I've fixed it and squashed it with your original kholidays commit.